infrastructure/variables.tf

```diff
@@ -117,7 +117,7 @@ variable "reports_cron" {
 # region incomplete envelopes monitoring
 
 variable "incomplete_envelopes_cron" {
-  default     = "0 0 * * * *"
+  default     = "0 */15 * * * *"
```

Comment:
I think this config is not used but worth keeping them in sync or completely get rid of this ?

Reply:
yes this was only used for ASE app. completely irrelevant right now

Reply:
we can remove all ASE related stuff on a PR dedicated to that subject.